### ISSUE TITLE
After setting minZoomLevel, the chart shifts to the bottom-right corner when zooming out

### ISSUE THREAD

**Issue by nontimebomala** [amcharts/amcharts5#1793]
**Question**

here is my settings:
```
const zoomableContainer = chartRoot.container.children.push(
    ZoomableContainer.new(chartRoot, {
      width: p100,
      height: p100,
      wheelable: true,
      pinchZoom: true,
      maxZoomLevel: 16,
      minZoomLevel: 0.2
    })
  );
```
1、When I zoom out, the chart moves to the bottom-right corner.
![image](https://github.com/user-attachments/assets/a77369eb-7db7-469b-a400-a1c30f1c51d9)

2、Following point 1, the chart then becomes undraggable.
3、When I zoom in, dragging becomes restricted (for example, I want to drag downward to make the content visible, but I can't drag downward).
![image](https://github.com/user-attachments/assets/e1814d9a-24bd-4fbb-8b74-39b388d08436)

**Environment (if applicable)**
V5

**Additional context**

**Comment by nontimebomala** [comment]
and i set the background to the ZoomableContainer,when i zoom out,its will be like this
![image](https://github.com/user-attachments/assets/045823ac-6473-46d9-8732-15fd20f55b00)

**Comment by zeroin** [comment]
Thanks, we will fix it in next release and will let you know. Meanwhile you can fix the issue by setting big maxPanOut for the zoomableContainer:

`maxPanOut: 10`

Do you use the latest version?
We will also fix the background issue.

**Comment by nontimebomala** [comment]
> Thanks, we will fix it in next release and will let you know. Meanwhile you can fix the issue by setting big maxPanOut for the zoomableContainer:
> 
> `maxPanOut: 10`
> 
> Do you use the latest version? We will also fix the background issue.

thanks for ur answer，i'm using the 5.10.7 version

**Comment by martynasma** [comment]
Fixed in [5.10.9](https://github.com/amcharts/amcharts5/releases/tag/5.10.9).

## [5.10.9] - 2024-12-20

### Added
- Added way to determine if text label was autotruncated (`label.text._display.trunacted`).
- Heikin Ashi indicator added to `StockChart`.

### Changed
- The following settings will carry over to `Label`'s HTML content if set: `fontFamily`, `fontSize`, `fontStyle`, `fontWeight`, `fontStyle`, `fontVariant`, `textDecoration`.
- When an `MapChart` is zoomed using mouse wheel it will no longer stop page scroll when its fully zoomed out.
- When arranging tooltips on an `XYChart`, will now pay attention to `tooltipContainerBounds.bottom` setting.

### Fixed
- `ZoomableContainer` was not zooming out properly if `minZoomLevel` was `< 1`.
- `PicturePattern` was ignoring `colorOpacity` setting.
- `PicturePattern` with `fit: "pattern"` set would sometimes not show actual image.
- "Measure" drawing tool of `StockChart` was not including one volume bar value.
- If a timezone was set on a chart and browser's locale was set to some non-English language, weekly grid was shown incorrecly on a `DateAxis`.
- Changing `text` on a `Label` with `oversizedBehavior="fit"` would sometimes not revert to original scale even if the new text would fit into max width.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
